client/src/components/Library.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+
+export default function Library({ library, isStatic }) {
+  return (
+    <BooksWrapper isStatic={isStatic}>
+      {library.map((book) => {
+        return (
+          <BookCard key={book.id} data-testid="library-book">
+            <img
+              src={book.volumeInfo.imageLinks.thumbnail}
+              alt="Book Cover"
+              width="102.4"
+              height="159.2"
+            />
+            <div>
```

Comment:
do you need div and p here? maybe one is enough?

Reply:
changed it to <p> only without the additional <div>